CommonConcepts/Plugins/Rhetos.Dom.DefaultConcepts/Persistence/FullTextSearchIntKeyInterceptor.cs

```diff
@@ -0,0 +1,136 @@
+﻿/*
+    Copyright (C) 2014 Omega software d.o.o.
+
+    This file is part of Rhetos.
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as
+    published by the Free Software Foundation, either version 3 of the
+    License, or (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.Data.Entity.Infrastructure.Interception;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Data.Common;
+using System.Data;
+using System.Text.RegularExpressions;
+
+namespace Rhetos.Dom.DefaultConcepts.Persistence
+{
+    /// <summary>
+    /// Based on http://www.entityframework.info/Home/FullTextSearch.
+    /// This interceptor modifies SQL query generated by FullTextSearchId function mapping in DatabaseExtensionFunctionsMapping.
+    /// </summary>
+    public class FullTextSearchIntKeyInterceptor : IDbCommandInterceptor

```

Comment:
Please try to modify `FullTextSearchInterceptor` class to handle both `InterceptFullTextSearchFunction` and `InterceptFullTextSearchFunctionIntKey`. `FullTextSearchIntKeyInterceptor` class might be removed since the implementation is similar to existing `FullTextSearchInterceptor`.